app/src/main/java/com/example/worldnews/domain/model/Error.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.worldnews.domain.model
+
+import android.content.Context
+
+data class Error(
+    val code: String,
+    val context: Context,
```

Comment:
Not a good idea to include any android related API in a data class

Reply:
Probably you meant following class, It's ok but the class name is misleading

Reply:
It has sth to do with the backend json object

Reply:
You don't need necesarrily to set a name based on the backed response. usually gson libraries provide an annotation to fetch it by a custom name, like: `@field:Json(name = "fieldName")` in [moshi](https://github.com/square/moshi)